src/javascript/_common/base/currency_base.js

```diff
@@ -109,6 +109,11 @@ const getCurrencyName = currency => getPropertyValue(CryptoConfig.get(), [curren
 
 const getMinPayout = currency => getPropertyValue(currencies_config, [currency, 'stake_default']);
 
+const getMaxTransfer = currency => {
+    const max_transfer = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits', 'max']);
```

Comment:
We already have this function `getMinTransfer`, I suggest grouping them together in one function such as `getTransferLimits(value, currency)` and pass in either `'min'` or `'max'` to get that instead of repeating logic in two functions

Reply:
then this would change to 
```
const transfer_limit = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits', value]);
```

Reply:
I first created the function `getMinMaxTransferLimits`, and then I thought not to complicate the function API and use another function to minify the changes. 

I'll go with your suggestion. fine by me :)